src/AzureStack/Subscriptions/Subscriptions/generate.ps1

```diff
@@ -0,0 +1,13 @@
+# Remove Old
+Remove-Item "$PSScriptRoot\Generated" -Force -Recurse -ErrorAction SilentlyContinue
```

Comment:
Removing old code can be achieved from your rest spec Readme.md using clear-output-folder: true